dragonfly/implementations/roach_daq_run_interface.py

```diff
@@ -507,7 +538,7 @@ def set_default_trigger(self):
         if self.default_trigger_dict == None:
             raise core.exceptions.DriplineGenericDAQError('No default trigger settings present')
         else:
-            self.configure_trigger(low_threshold=self.default_trigger_dict['snr_threshold'], high_threshold=self.default_trigger_dict['snr_high_threshold'], n_triggers=self.default_trigger_dict['n_triggers'])
+            self.configure_trigger(threshold=self.default_trigger_dict['threshold'], high_threshold=self.default_trigger_dict['high_threshold'], n_triggers=self.default_trigger_dict['n_triggers'])
```

Comment:
is this dictionary defined in this class? if you change a field name, don't you need to change it elsewhere?